nucliadb_node/binding/src/lib.rs

```diff
@@ -59,6 +59,7 @@ impl NodeReader {
 
     pub fn get_shard<'p>(&mut self, shard_id: RawProtos, py: Python<'p>) -> PyResult<&'p PyAny> {
         let shard_id = ShardId::decode(&mut Cursor::new(shard_id)).unwrap();
+        self.reader.load_shard(&shard_id);
```

Comment:
I wonder if it is not more ergonomic to nest the `self.reader.load_shard(&shard_id)` call in methods that take a `shard_id` as parameter (such as `search`, `suggest` and so on) because it looks like error-prone and easy to forget that `load_shard` has to be called before-hand.

Reply:
*UPDATE*: I just saw the other method named `shard_loading`.  You can ignore my comment.